src/unit_tests/dataset_tests.cpp

```diff
@@ -55,7 +55,7 @@ BOOST_AUTO_TEST_CASE(get_block_size)
 
     ld.get_block_size(locked_dataset::WARPED, 1, &width, &height);
     BOOST_TEST(width == 512);
-    BOOST_TEST(height == 128);
+    BOOST_TEST(height == 512);
```

Comment:
So weird it's been 128 before.

Reply:
I also saw this error, but I didn't have time to find the source.  Thanks.